game/static/game/css/forum.css

```
@@ -133,8 +133,8 @@
     padding: 0 2rem;
     min-height: 80px;
 
-    background: rgba(10,10,20,0.95);
-    border-bottom: 1px solid rgba(255,255,255,0.08);
+    background: var(--nav-bg);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)